config/rbac/role.yaml

```diff
@@ -186,6 +186,30 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - instancetype.kubevirt.io
+  resources:
+  - virtualmachineclusterinstancetypes
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - instancetype.kubevirt.io
+  resources:
+  - virtualmachineclusterpreferences
```

Comment:
You could add this on line 193 and avoid the duplication of the verbs.

Reply:
These files are actually generated so this isn't actually possible unfortunately.

Reply:
ACK!